packages/platform-sdk-generator/src/generatePlatformSdkv2.ts

```diff
@@ -290,7 +290,7 @@ const BASE_PACKAGE_JSON = {
       "com.palantir.foundry.api:api-gateway": {
         "minVersion": "1.824.0",
         "maxVersion": "1.x.x",
-        "optional": true,
+        "optional": false,
```

Comment:
I explicitly declared as false, not sure if we need this line at all though. I looked at internal examples and saw it done both ways